lib/web/apiserver.go

```diff
@@ -172,6 +175,11 @@ type Handler struct {
 	// an authenticated websocket so unauthenticated sockets dont get left
 	// open.
 	wsIODeadline time.Duration
+
+	// featureWatcherStop is a channel used to emit a stop signal to the
+	// license watcher goroutine
```

Comment:
Is this a license watcher or a feature watcher? (or both?)

Reply:
It's a **feature** watcher, I mixed the two in some names. Updated in https://github.com/gravitational/teleport/pull/45979/commits/a5561d312bcae446fd67eecb16ae3952a35512ce, thanks.

I also updated all names to `feature` (singular), as some used to say `features` (eg `startFeaturesWatcher`).